utils/qemu/patches/0008-meson-python-fix.patch

```diff
@@ -0,0 +1,51 @@
+--- a/meson.build
```

Comment:
This patch is no longer needed.

Reply:
Yes, i removed that patch and it is building. Note i'm not yet got anything working. Seems at lease x86 need libbpf now.

Reply:
Qemu compilled. Will update my router soon...